front/app/containers/Admin/projectFolders/containers/settings/ProjectFolderForm/index.tsx

```diff
@@ -86,20 +132,22 @@ const ProjectFolderForm = ({ mode, projectFolderId }: Props) => {
   useEffect(() => {
     (async () => {
       if (mode === 'edit' && !isNilOrError(projectFolderImagesRemote)) {
-        const imagePromises = projectFolderImagesRemote.data.map((img) =>
-          img.attributes.versions.medium
-            ? convertUrlToUploadFile(
-                img.attributes.versions.medium,
-                img.id,
-                null
-              )
-            : new Promise<null>((resolve) => resolve(null))
-        );
+        const imagePromises = projectFolderImagesRemote.data.map((img) => {
+          const url =
+            // On the homepage, we use the large version when screen
+            // width > 1200px, so this shows a more realistic preview
+            previewDevice === 'desktop'
+              ? img.attributes.versions.large
+              : img.attributes.versions.medium;
```

Comment:
This is the change in functionality, the rest is just shuffling things around.

Reply:
I'm not sure it's consistent with FrontOffice behavior https://github.com/CitizenLabDotCo/citizenlab/blob/e480cb4655f7b40333abc77fa50bb8f7d06d836c/front/app/components/ProjectCard/index.tsx#L521 (it seems it'll use `large` for tablet too).

Also, it can diverge in the future, and the preview will be not the same as the real card. It seems we've fixed a few similar issues recently.

So, I would propose to share this logic between two components (BackOffice and FrontOffice). But not critical, up to you.

Reply:
Yes indeed, I was confused because we use the same wide card for both desktop and tablet, but use the large version for desktop, medium version for tablet.

Thanks for pointing out.

It's really hard to get it exactly right for all cases (e.g. a project card shown inside a folder will be smaller). Not sure if it's even possible to cover these all at the moment...

Reply:
> Not sure if it's even possible to cover these all at the moment

I guess it'll be easier if we share some code.

Reply:
Yes, working on something. I'll tag you when it's up 👍